ci-operator/config/openshift-qe/ocpqe-security-tools/openshift-qe-ocpqe-security-tools-main__dast.yaml

```diff
@@ -0,0 +1,106 @@
+      # Install Go. This version will only be used for unit tests. The
+      # official builds are done via their own container image Dockerfiles
+      RUN rm -rf /usr/local/go && \
+          curl -fsSL https://go.dev/dl/go${GOVERSION}.linux-amd64.tar.gz | tar -C /usr/local -xzf -
+      ENV PATH=$PATH:/usr/local/go/bin
+      RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 && \
+          chmod 700 get_helm.sh && \
+          ./get_helm.sh
+      RUN pip install virtualenv
+images:
+- dockerfile_literal: |
+    FROM src
+    COPY oc /usr/bin/oc
+    RUN ln -s /usr/bin/oc /usr/bin/kubectl
+  from: src
+  inputs:
+    base:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: ocp-qe-ci-dast
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.17"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: security-first-half
+  cron: 0 16 5 * *
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    env:
+      API_URL_LIST: admissionregistration.k8s.io/v1 apiextensions.k8s.io/v1 apiregistration.k8s.io/v1
+        apiserver.openshift.io/v1 apps.openshift.io/v1 apps/v1 authentication.k8s.io/v1
+        authorization.k8s.io/v1 authorization.openshift.io/v1 autoscaling.openshift.io/v1
+        autoscaling.openshift.io/v1beta1 autoscaling/v1 autoscaling/v2 batch/v1 build.openshift.io/v1
+        certificates.k8s.io/v1 cloud.network.openshift.io/v1 cloudcredential.openshift.io/v1
+        config.openshift.io/v1 console.openshift.io/v1 console.openshift.io/v1alpha1
+        controlplane.operator.openshift.io/v1alpha1 coordination.k8s.io/v1 discovery.k8s.io/v1
+        events.k8s.io/v1 flowcontrol.apiserver.k8s.io/v1 flowcontrol.apiserver.k8s.io/v1beta3
+        helm.openshift.io/v1beta1 image.openshift.io/v1 imageregistry.operator.openshift.io/v1
+        infrastructure.cluster.x-k8s.io/v1alpha5 infrastructure.cluster.x-k8s.io/v1beta1
+        ingress.operator.openshift.io/v1 ipam.cluster.x-k8s.io/v1alpha1 ipam.cluster.x-k8s.io/v1beta1
+        k8s.cni.cncf.io/v1 k8s.ovn.org/v1 logging.openshift.io/v1 logging.openshift.io/v1alpha1
+        loki.grafana.com/v1 machine.openshift.io/v1 machine.openshift.io/v1beta1 machineconfiguration.openshift.io/v1
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - ref: openshift-qe-ocpqe-security-tools-dast
+    workflow: openshift-qe-installer-aws
+  timeout: 4h0m0s
+- always_run: false
+  as: security-test
+  steps:
+    env:
+      API_URL_LIST: admissionregistration.k8s.io/v1
+    test:
+    - ref: openshift-qe-ocpqe-security-tools-dast
+  timeout: 1h0m0s
+- as: security-second-half
+  cron: 0 16 15 * *
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    env:
+      API_URL_LIST: metal3.io/v1alpha1 metrics.k8s.io/v1beta1 migration.k8s.io/v1alpha1
```

Comment:
@paigerube14 Could this be  dynamically generated by another ref to pass it on  this one?

Reply:
I think we can definitely do that, was hoping to get the base runs and functionality in and then add that in as a next step. This will also help on other cloud providers if there are other apis that are available 
Created this jira to track: [Auto generate the api list](https://issues.redhat.com/browse/OCPQE-23789)